.github/workflows/release-asset-publish.yml

```diff
@@ -0,0 +1,42 @@
+name: Publish release .jar
+
+on:
+  push:
+    tags:
+      - '*'
```

Comment:
shouldn't we restrict the publishing from the master branch only? (maybe it's not a problem since we restrict the permission to push tags already)

Reply:
We restricted by tags. Usually any new tag == new release. So...

Reply:
I mean that potentially it's possible to create a tag anywhere, so you trigger artifact publishing. It was a question about whether we need "a protection from the fool" or not (or we already have it).

Reply:
@AntoxaAntoxic you can't create tag without release publish. Also you can't publish release with the same tag twice.